### DISCUSSION TITLE
Support for volumetric data

### DISCUSSION THREAD

**Discussion by MaximilianHoffmann** [miniscope/minian#185] [Ideas]
Have you thought about extending the package to be able to process 4D data? I understand, that it was created for miniscope data, but I think it could be very useful for 1p-volumetric imaging techniques, too. Would this be of interest?

**Comment by phildong** [comment]
I'm curious what extra steps/analysis you have in mind for 4D data? alignment across multiple z-plane?

So I think personally I would not be able to have the time to work on this, but I'd love to see some PR and I'd be happy to provide any help I can!
